package.json

```diff
@@ -105,9 +105,9 @@
     "can-bind-to-host": "^1.1.1",
     "commander": "^9.0.0",
     "global": "^4.4.0",
-    "jest-playwright-preset": "^1.7.2",
+    "jest-playwright-preset": "^2.0.0",
```

Comment:
By expanding the range, existing projects can still use Jest 27, and new projects can benefit of using Jest 28.

The unfortunate breaking event would be in projects that use Jest 27 and the test runner works fine, but someone decides to delete their package lock and rerun install, which then would result in a broken test runner. 

If people want to use Jest 27 they can use yarn resolutions or npm overrides to downgrade `jest-playwright-preset` to `^1.7.2`. This should probably be added to the FAQ in README to guide people to still have Jest 27 support.

```suggestion
    "jest-playwright-preset": "^1.7.2 || ^2.0.0",
```

Reply:
Will add to README FAQ